### PR TITLE
Open gmail.com in the default browser

### PR DESCRIPTION
**Pull request by tonka3000**
## Description

Open https://gmail.com in the default browser

## Type of change

- [x] New script command
- [ ] Bug fix
- [ ] Improvement of an existing script
- [ ] Documentation update
- [ ] Toolkit change
- [ ] Other (Specify)

## Screenshot

gmail-script-command

## Dependencies / Requirements

no dependencies

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

**Comment by tonka3000** [comment]
@thomaspaulmann

File: commands/communication/gmail/google-gmail.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# @raycast.title GMail
```
Comment (thomaspaulmann) [inline commands/communication/gmail/google-gmail.sh:3]:
```suggestion
# @raycast.title Gmail
```

Comment (thomaspaulmann) [reply commands/communication/gmail/google-gmail.sh:3]: Lets follow the product name: https://gsuite.google.com/products/gmail/

**Review by thomaspaulmann — APPROVED** [review]
Nice and simple one!

**Pull request merged**